prs/ChicagoStyleSausagePizza.ts

```diff
@@ -0,0 +1,13 @@
+class ChicagoStyleSausagePizza extends Pizza {
+    public ChicagoStyleSausagePizza() {
+        this.pizzaType = "Chicago Style Deep Dish Cheese Pizza";
+        this.doughType = "Extra Thick Crust Dough";
+        this.sauceType = "Plum Tomato Sauce";
```

Comment:
Also, adding the `Type` suffix implies that the underlying data type should be mapping to an enum or similar.  Consider what the intent and update the naming to match expectations.